client/src/partials/templates/modals/findEntity.modal.html

```diff
@@ -1,69 +1,79 @@
-<div class="modal-header">
-  <h4>
-    <i class="glyphicon glyphicon-search"></i>
-    {{ 'VOUCHERS.COMPLEX.ENTITY' | translate }}
-    <span
-      ng-if="$ctrl.result.uuid"
-      ng-click="$ctrl.refresh()"
-      class="btn btn-default pull-right">
-      <i class="glyphicon glyphicon-refresh"></i>
-    </span>
-  </h4>
-</div>
+<ng-form
+  name="EntityForm"
+  autocomplete="off"
+  autocorrect="off"
+  autocapitalize="off"
+  data-modal="entity"
+  novalidate>
 
-<div class="modal-body clearfix">
+  <div class="modal-header">
+    <h4>
+      <span class="glyphicon glyphicon-search"></span>
+      {{ 'VOUCHERS.COMPLEX.ENTITY' | translate }}
 
-  <div class="form-group">
+      <span
+        ng-show="$ctrl.result.uuid"
+        ng-click="$ctrl.refresh()"
+        class="btn btn-default btn-sm pull-right">
```

Comment:
I think it is generally good practice to use `float` on containers and then have the content within the containers placed to ensure that content isn't corrupted by the page flow change. I can't immediately find any good documentation to back this up though. 

As this is a span is it possible/ potentially easier to use `text-align : right;` ?